cypress/integration/pages/article/amp.js

```diff
@@ -164,5 +164,26 @@ Object.keys(config)
           `https://www.bbc.com${config[service].pageTypes.articles}`,
         );
       });
+
+      describe('Assert for broken links', () => {
+        it('should not contain any broken header links - no 404s links', () => {
+          cy.get('header a').each(element => {
```

Comment:
Can this be a cypress command so it can be reused in all page types? (untill https://github.com/bbc/simorgh/pull/3053)

Reply:
yeah sure.